learning-team/2020-08/2020-08-20.md

```diff
@@ -0,0 +1,49 @@
+# August 20, 2020
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [ ] Amy Lam (AL)
+- [ ] Chris Manson (CM)
+- [ ] Isaac Lee (IL)
+- [ ] Jared Galanis (JG)
+- [ ] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [ ] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [] Your name here (INITIALS)
+
+## Weekly review
+
+### Review action items (weekly)
+See: [Action items board](https://github.com/orgs/ember-learn/projects/47)
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/19)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
```

Comment:
```suggestion
### reader feedback regarding https://guides.emberjs.com/release/components/looping-through-lists/
>>> JoxtalToday at 1:34 PM
So reading https://guides.emberjs.com/release/components/looping-through-lists/ I tripped a little.
It mentions the need to convert an array into an EmberArray, and links to https://api.emberjs.com/ember/release/classes/EmberArray.
Alright, it's an observer friendly wrapper so that things will automatically be notified when the array changes. Cool.
But then I start looking for the method used in the example, this.messages.pushObject to see what it does.
But I can't find it. I start to scratch my head because it's not a part of EmberArray. :thinking: 
So then I look at the import { A } from '@ember/array and see that EmberArray belongs to that module, so I click the link to https://api.emberjs.com/ember/3.20/modules/@ember%2Farray and find a reference to A!
Great, I'll check that out.
There I read A() is not needed if EmberENV.EXTEND_PROTOTYPES is true (the default value). However, it is recommended that you use A() when creating addons for ember or when you can not guarantee that EmberENV.EXTEND_PROTOTYPES will be true..
Hmm, okay, so I shouldn't need to use it?
I'm confused now and I still haven't found docs for .pushObject. It was not a part of A.
Now I'm wondering when and if I need to use EmberArray, if I should or should not use A, what EmberENV.EXTEND_PROTOTYPES does, and if this was just a rabbit hole of my own making because I'm slow to learn.
A little later I happen to catch that the A function returns an Ember.NativeArray and not an EmberArray and it's in https://api.emberjs.com/ember/3.20/classes/Ember.NativeArray where I find pushObject!
Perhaps this little stumble could have been avoided somehow? :man_shrugging:

<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
```